attestation-container/go.mod

```diff
@@ -0,0 +1,16 @@
+module microsoft/attestation-container
```

Comment:
If we want to publish the module, we can change it to `github.com/microsoft/CCF/attestation-container`.
https://go.dev/doc/tutorial/create-module#start

But I think it's fine to do it after creating attestation container repository (It would be `github.com/microsoft/attestation-container` ).

Reply:
We mainly intend to release this as a container, not as a module at this time. But it's not to have the option for later.